news/add_expense-db_option_to_presentation_adder.rst

```diff
@@ -0,0 +1,30 @@
+**Added:**
+* Added --expense-db flag to a_presentation adder. If specified, it adds 
```

Comment:
don't need "added" as it is part of the "added" section.  I think I would write it as 
```Added:
* functionality that allows a user to specify a different database for the expense and presentation items when running a_presentation.
```